person.rb

```diff
@@ -1,26 +1,21 @@
 # person.rb
 
-#
-# Interface Class Nameable, for decorator design
-#
-class Nameable
-  def correct_name
-    raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
-  end
-end
+require_relative 'nameable'
 
 #
 # Class representing a person in the library application
 #
 class Person < Nameable
   attr_accessor :name, :age
   attr_writer :id
+  attr_reader :rentals
```

Comment:
- [ ] to make this relationship work Getters and Setters for @rentals should be present

Reply:
Thank you. changes done!